tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/swtbot/SWTBotMenuHelper.java

```diff
@@ -64,4 +69,21 @@ public SWTBotMenu findMenu(final SWTBotMenu parentMenu,
             return new SWTBotMenu(menuItem);
         }
     }
+
+    public SWTBotMenu findWorkbenchMenu(final SWTWorkbenchBot bot, final String menuString) {
```

Comment:
It might be the case to write a well-documenting Javadoc here, in particular, the problems it fixes